.gitignore

```diff
@@ -282,4 +280,4 @@ cython_debug/
 
 # Emacs
 *~
-\#*\#
\ No newline at end of file
+\#*\#
```

Comment:
you need to add `node_modules/` to gitignore
and run `npm install` in action.yml

Reply:
I don't think that actions allow this. The only entrypoint that I can provide is a single js file.
Also, adding `node_modules` is recommended by github docs
https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#commit-tag-and-push-your-action-to-github

Reply:
They do
`git add action.yml index.js node_modules/* package.json package-lock.json README.md`

Reply:
and then a few steps after they said
```Checking in your node_modules directory can cause problems. As an alternative, you can use a tool called @vercel/ncc to compile your code and modules into one file used for distribution.```
also we can use `rollup` to bundle this script and it dependencies to one file

Reply:
am not sure how it will be when we have to update some of dependencies, I've never commit `node_modules`